src/main/scala/chisel3/util/Reg.scala

```diff
@@ -42,7 +42,7 @@ object ShiftRegister
     * val regDelayTwo = ShiftRegister(nextVal, 2, ena)
     * }}}
     */
-  def apply[T <: Data](in: T, n: Int, en: Bool = true.B): T = ShiftRegisters(in, n, en).lastOption.getOrElse(in)
+  def apply[T <: Data](in: T, n: Int, en: Bool = true.B, flatten: Boolean = true): T = ShiftRegisters(in, n, en, flatten).lastOption.getOrElse(in)
```

Comment:
I think "inline" is a better name for this parameter, it's more industry standard